src/main/java/com/medals/medalsbackend/service/user/TrainerService.java

```
@@ -99,4 +103,20 @@ public void updateTrainer(Long trainerId, TrainerDto trainerDto) {
         Trainer savedTrainer = (Trainer) userEntityService.update(objectMapper.convertValue(trainerDto, Trainer.class));
         trainerWebsocketMessageService.sendTrainerUpdate(objectMapper.convertValue(savedTrainer, TrainerDto.class));
     }
+
+    public void inviteAthlete(AthleteSearchDto athleteSearchDto) throws AthleteNotFoundException {
+        Long athleteId = athleteSearchDto.getAthleteId();
+        Athlete inviteAthlete = (Athlete) userEntityService.findById(athleteId).orElseThrow(() -> AthleteNotFoundException.fromAthleteId(athleteId));
+        log.info("Executing invite athlete {}", inviteAthlete);
+
+        String trainerName = userEntityService.findById(athleteSearchDto.getTrainerId())
+                .map(user -> user.getFirstName() + " " + user.getLastName())
+                .orElse("Unknown Trainer");
```

This should throw an error if no user was found and not work on further